app/src/editor/canvas/components/Module.jsx

```diff
@@ -129,14 +130,15 @@ class CanvasModule extends React.PureComponent {
         const isRunning = canvas.state === RunStates.Running
         const isResizable = isModuleResizable(module)
 
+        const moduleSpecificStyles = [ModuleStyles[module.jsModule], ModuleStyles[module.widget]]
         return maybeConnectDragging((
             /* eslint-disable-next-line max-len */
             /* eslint-disable jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-tabindex */
             <div
                 role="rowgroup"
                 tabIndex="0"
                 onFocus={() => api.selectModule({ hash: module.hash })}
-                className={cx(styles.Module, {
+                className={cx(styles.CanvasModule, ModuleStyles.ModuleBase, ...moduleSpecificStyles, {
```

Comment:
Could just write out `ModuleStyles[module.jsModule], ModuleStyles[module.widget]` here? Or `moduleSpecificStyles` is for verbosity?

Reply:
Yeah I think the line got too long